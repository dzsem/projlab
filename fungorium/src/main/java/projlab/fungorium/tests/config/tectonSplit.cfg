# Tecton Split
# ------------------------------------------
# Init state
# ------------------------------------------
# ID: 0
ADD TECTON
# 1
ADD TECTON
REGISTER NEIGHBOUR 0 1
# 2
ADD MUSHROOMBODY 0 1
# 3
ADD MUSHROOMSPORE 1
# 4 
ADD MUSHROOMTHREAD 1 0
# 5
ADD MUSHROOMTHREAD 1 1
REGISTER CONNECTION 4 5
# 6
ADD INSECT 1 0

SET TECTONSPLITCHANCE 100