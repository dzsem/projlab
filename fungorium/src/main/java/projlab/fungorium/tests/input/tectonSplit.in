# Tecton Split
# ------------------------------------------
# Init state
# ------------------------------------------
# ID: 1
ADD TECTON
# 2
ADD TECTON
REGISTER NEIGHBOUR 1 2
# 3
ADD MUSHROOMBODY 1 1
# 4
ADD MUSHROOMSPORE 1 1
# 5 
ADD MUSHROOMTHREAD 1 1
# 6
ADD MUSHROOMTHREAD 1 2
REGISTER CONNECTION 5 6
# 7
ADD INSECT 1 1
# Input
# ------------------------------------------
NEXTROUND
# Print states
# ------------------------------------------
LIST TECTON