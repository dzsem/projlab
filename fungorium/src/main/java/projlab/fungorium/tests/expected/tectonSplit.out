TECTON 0 2 -1 1 0
TECTON 1 2 2 1 1
TECTON 7 1 -1 0 0